src/mapP.js

```diff
@@ -1,8 +1,30 @@
 const curry = require('ramda/src/curry')
 const map   = require('ramda/src/map')
 
-// mapP :: Functor f => (a -> Promise b) -> f a -> Promise f b
-const mapP = (f, list) =>
+const mapArrayP = (f, list) =>
   Promise.all(map(f, list))
 
+const mapObjectP = (f, list) => {
+  const result = {}
+  const promises = []
+
+  for (const key in list) {
+    const p = f(list[key]).then(val => { result[key] = val })
```

Comment:
I think it'd be helpful to go ahead and accept binary morphisms `v -> k -> Promise v` so we can have access to the key if needed, like `f(obj[key], key)`.

Reply:
Thought about this, but it think this goes against the spirit of ramda. They have a seperate `mapObjectIndexed` for this. I think forcing a unary function is safer.